functions/world.js

```diff
@@ -158,7 +158,7 @@ exports.updateWorld = functions.https.onCall(async (data, context) => {
     ...(!isNil(requiresDateOfBirth) && { requiresDateOfBirth }),
     ...(!isNil(rooms) && { rooms }),
     ...(!isNil(showNametags) && { showNametags }),
-    ...(!isNil(showSchedule) && { showSchedule }),
+    ...{ showSchedule: isNil(showSchedule) ? true : showSchedule },
```

Comment:
Are we 100% sure that we want to default to `true` when `showSchedule` is `null` or `undefined`? Also if we do, we should probably use a `DEFAULT_` value here.

Reply:
Yes, that was the requirement (it kind of hinted here https://github.com/sparkletown/internal-sparkle-issues/issues/1240#issuecomment-958066249 but was explicitly said in a meeting) .

The `DEFAULT_` I think is used in the TS files, not sure if there is anything like it in the cloud functions, I mean, even the old `venue.showSchedule` used a `true` literal 🤷

Reply:
I have one or two PRs creating a new file `./src/settings/worldDefaultSettings.ts` and after we have all we need there it should be simple enough to transfer the knowledge to an analogue on the BE.